enterprise/server/WindowServerCore/Dockerfile

```diff
@@ -18,11 +18,16 @@ COPY setup .
 # Exposing the port 9900 at which Bridge service will listen for incoming connection requests from cache processes.
 # Exposing the port 8260 at which Bridge service will listen for incoming Management connection requests (NCache Manager and Tools).
```

Comment:
if you update the port that are exposed then update the comments too

Reply:
explained in  comments